.tekton/siteconfig-acm-214-pull-request.yaml

```diff
@@ -83,7 +83,7 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
```

Comment:
Updating the pipeline defaults in `pipelineSpec.params`, while it works, is not a best practice for setting the build parameters. These should be set as values under `spec.params`.

Reply:
Thanks @dhaiducek! I was already setting the correct values in `spec.params`. Reverted the change!

Reply:
@sakhoury While your `main` pipeline specs have the hermetic and build-source-image params, the `release-2.14` specs do not, so the build will pass in `main` but fail in 2.14.

Reply:
Thanks @dhaiducek - PR to address that: https://github.com/stolostron/siteconfig/pull/266.